manifests/init.pp

```diff
@@ -112,7 +112,7 @@
   ) inherits certs::params {
 
   if $server_cert {
-    validate_file_exists($server_cert, $server_cert_req, $server_key, $server_ca_cert)
+    validate_absolute_path([$server_cert, $server_cert_req, $server_key, $server_ca_cert])
```

Comment:
This is not the same: we need to check that the file really exists. On the other hand, even more checks happen now in https://github.com/Katello/katello-installer/blob/master/hooks/pre/20-certs_update.rb#L62, so I don't have that much issues then